libs/datagrid/src/lib/cells/order-type-cell.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { MouseEvent } from 'react';
+import { useMemo } from 'react';
+import { useCallback } from 'react';
+import get from 'lodash/get';
+import { t } from '@vegaprotocol/i18n';
+import * as Schema from '@vegaprotocol/types';
+
+interface OrderTypeCellProps {
+  value?: Schema.OrderType;
+  data?: {
+    id?: string;
+    marketId?: string;
+    market?: { id: string };
+    peggedOrder?: any;
+    liquidityProvision?: any;
```

Comment:
chore (blocking): Why not remove the `any`s with

```
peggedOrder: Order['peggedOrder']
liquidityProvision: Order['liquidityProvision']
```

Reply:
Or actually

```
data?: Order
```

Reply:
updated